### PR TITLE
Issue #13345: Enable LambdaParameterNameCheckExamplesTest

### PR DESCRIPTION
**Pull request by TarunAlapati-git**
Issue: https://github.com/checkstyle/checkstyle/issues/13345

Development changes :

* Modified the text files Example 1 and 2 to java input files
* ensured checkstyle is succeeded
* removed unwanted // OK comment

### CODE REVIEW

**Comment by romani** [comment]
GitHub, generate website

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/lambdaparametername/Example1.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+package com.puppycrawl.tools.checkstyle.checks.naming.lambdaparametername;
+
+import java.util.function.Function;
+
+// xdoc section -- start
+class Example1 {
+  Function<String, String> function1 = s -> s.toLowerCase();
+  Function<String, String> function2 =
+          S -> S.toLowerCase(); // violation, name 'S'
```
Comment (romani) [inline src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/lambdaparametername/Example1.java:17]: Please keep comments as simple `// violation `
To match other Checks.
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/7d927bc_2023165810/checks/naming/localvariablename.html#Example1-code

https://github.com/checkstyle/checkstyle/blob/7d927bc9ce2cb9fcca250f186ef1645da8da5b08/src/xdocs-examples/java/com/puppycrawl/tools/checkstyle/checks/naming/LocalVariableNameCheckExamplesTest.java#L28

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/lambdaparametername/Example1.java (hunk 1 above)
Comment (TarunAlapati-git) [reply src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/naming/lambdaparametername/Example1.java:17]: Sure, will change it

**Pull request closed without merging**